content/blog/2022-05-24-MLEM-release.md

```diff
@@ -0,0 +1,324 @@
+💾 Updating deployment at my-service.mlem
+🛠 Releasing app mlem-quick-start formation
+💾 Updating deployment at my-service.mlem
+✅  Service mlem-quick-start is up. You can check it out at https://mlem-quick-start.herokuapp.com/
+```
+
+Since MLEM is both CLI-first and API-first tool, you can productionize your
+models just as easy with Python API:
+
+```python
+$ python
+>>> from mlem.api import serve, pack, deploy
+```
+
+# Git-native model registry
+
+[versions]: /doc/use-cases/versioning-data-and-model-files
+[mp]: /doc/start/metrics-parameters-plots
+[experiments]: /doc/user-guide/experiment-management
+[gto]: https://github.com/iterative/gto
+[mlem]: https://mlem.ai/
+[modeling process]: /doc/start/data-pipelines
+[remote storage]: /doc/command-reference/remote
+[sharing]: /doc/start/data-and-model-access
+[via cml]: https://cml.dev/doc/cml-with-dvc
+[gitops]: https://www.gitops.tech/
+
+MLEM is a core building block for a Git-based ML model registry, together with
+other Iterative tools, like GTO and DVC.
+
+ML model registries give your team key capabilities:
+
+- Collect and organize model [versions] from different sources effectively,
+  preserving their data provenance and lineage information.
+- Share metadata including [metrics and plots][mp] to help use and evaluate
+  models.
+- A standard interface to access all your ML artifacts, from early-stage
+  [experiments] to production-ready models.
+- Deploy specific models on different environments (dev, shadow, prod, etc.)
+  without touching the applications that consume them.
+- For security, control who can manage models, and audit their usage trails.
+
+Many of these benefits are built into DVC: Your [modeling process] and
+[performance data][mp] become **codified** in Git-based <abbr>DVC
+repositories</abbr>, making it possible to reproduce and manage models with
+standard Git workflows (along with code). Large model files are stored
+separately and efficiently, and can be pushed to [remote storage] -- a scalable
+access point for [sharing].
+
+To make a Git-native registry (on top of DVC or not), one option is to use [GTO]
+(Git Tag Ops). It tags ML model releases and promotions, and links them to
+artifacts in the repo using versioned annotations. This creates abstractions for
+your models, which lets you **manage their lifecycle** freely and directly from
+Git.
+
+```shell
+$ gto show
+╒══════════════════════╤══════════╤════════╤═════════╕
+│ name                 │ latest   │ #stage │ #prod   │
+╞══════════════════════╪══════════╪════════╪═════════╡
+│ pet-face-recognition │ v3.1.0   │ -      │ v3.0.0  │
+│ mlem-blep-classifier │ v0.4.1   │ v0.4.1 │ -       │
+│ dog-bark-translator  │ v0.0.1   │ -      │ v0.0.1  │
+╘══════════════════════╧══════════╧════════╧═════════╛
+
+$ mlem apply dog-bark-translator ./short-dog-phrase.wav
+🐶🚀🎉
+```
+
+# What next?
+
+**Star** [MLEM on GitHub](https://github.com/iterative/mlem) and let us know
```

Comment:
Sorry I didn't communicate that clearly @aguschin, per also Ivan's comment use the Star emoji and then Star 

⭐️ Star [MLEM on GitHub](https://github.com/iterative/mlem) and let us know